tests/scripts/all.sh

```diff
@@ -787,6 +787,24 @@ component_test_rsa_no_crt () {
     if_build_succeeded tests/compat.sh -t RSA
 }
 
+component_test_no_ctr_drbg () {
+    msg "build: Full minus CTR_DRBG"
+    scripts/config.pl full
+    scripts/config.pl unset MBEDTLS_CTR_DRBG_C
+    scripts/config.pl unset MBEDTLS_PSA_CRYPTO_C # requires CTR_DRBG
+    scripts/config.pl unset MBEDTLS_PSA_CRYPTO_STORAGE_C # requires PSA Crypto
+    scripts/config.pl unset MBEDTLS_PSA_CRYPTO_SE_C # requires PSA Crypto
+    scripts/config.pl unset MBEDTLS_USE_PSA_CRYPTO # requires PSA Crypto
```

Comment:
There's no PSA in 2.16. There's a tension between keeping backports aligned and keeping them free of things that don't belong there. When it comes to PSA stuff, I think 2.16 shouldn't have any.

Reply:
Good point. Keeping those was not intentional, I agree they don't belong in 2.16. I'll remove them.